docs/src/main/sphinx/language/overview.rst

```diff
@@ -0,0 +1,133 @@
+=====================
+SQL statement support
+=====================
+
+The SQL statement support in Trino can be categorized into a number of topics.
+Numerous statements are part of the core engine and therefore available in all
+use cases. For example, you can always set session properties or inspect an
+explain plan and perform other actions with the :ref:`globally available
+statements <sql-globally-available>`.
+
+On the other hand the details and architecture of the connected data sources may
+prevent some SQL functionality. For example, if the data source does not support
+any write operations, Trino can not implement support for SQL statements like
+:doc:`/sql/delete`.
+
+Similarly if the underlying system does not have any security concepts, SQL
+statements like :doc:`/sql/create-role` can not provided by Trino and the
+connector, or they are simply not implemented in the connector.
+
+The categories of these different topics are related to :ref:`read operations
+<sql-read-operations>`, :ref:`write operations <sql-write-operations>`,
+:ref:`security operations <sql-security-operations>` and :ref:`transactions
+<sql-security-transactions>`.
+
+.. _sql-globally-available:
+
+Globally available statements
+-----------------------------
+
+The following statements are implemented in the core engine and available with
```

Comment:
We could also break this up a bit into the following 

* session stuff
* prepared statement stuff
* query/explain stuff 

but I am not sure its worth doing that